2022-12-05 10:22:45,777 INFO    StreamThr :20488 [internal.py:wandb_internal():88] W&B internal server running at pid: 20488, started at: 2022-12-05 10:22:45.776757
2022-12-05 10:22:45,780 DEBUG   HandlerThread:20488 [handler.py:handle_request():138] handle_request: status
2022-12-05 10:22:45,785 DEBUG   SenderThread:20488 [sender.py:send_request():317] send_request: status
2022-12-05 10:22:45,788 INFO    WriterThread:20488 [datastore.py:open_for_write():75] open: E:\yolov7-main\wandb\offline-run-20221205_102245-1vnl2o9e\run-1vnl2o9e.wandb
2022-12-05 10:22:45,789 DEBUG   HandlerThread:20488 [handler.py:handle_request():138] handle_request: run_start
2022-12-05 10:22:45,822 DEBUG   HandlerThread:20488 [meta.py:__init__():34] meta init
2022-12-05 10:22:45,822 DEBUG   HandlerThread:20488 [meta.py:__init__():49] meta init done
2022-12-05 10:22:45,822 DEBUG   HandlerThread:20488 [meta.py:probe():215] probe
2022-12-05 10:22:45,832 DEBUG   HandlerThread:20488 [git.py:repo():40] git repository is invalid
2022-12-05 10:22:45,832 DEBUG   HandlerThread:20488 [meta.py:_save_pip():53] save pip
2022-12-05 10:22:45,833 DEBUG   HandlerThread:20488 [meta.py:_save_pip():67] save pip done
2022-12-05 10:22:45,833 DEBUG   HandlerThread:20488 [meta.py:probe():253] probe done
